pkg/config/happy_config.go

```diff
@@ -17,7 +17,7 @@ type Environment struct {
 	SecretARN          string     `yaml:"secret_arn"`
 	TerraformDirectory string     `yaml:"terraform_directory"`
 	DeleteProtected    bool       `yaml:"delete_protected"`
-	AutoRunMigration   bool       `yaml:"auto_run_migration"`
+	AutoRunMigrations  bool       `yaml:"auto_run_migrations"`
```

Comment:
Do we have a test that reads a config file and proves that all the values are set correctly?

Reply:
there should be, but probably missing this. updating

Reply:
added to the test